crd-generator/pom.xml

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) 2015 Red Hat, Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.fabric8</groupId>
+    <artifactId>kubernetes-client-project</artifactId>
+    <version>5.1-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>crd-generator</artifactId>
+  <packaging>jar</packaging>
+  <name>Fabric8 :: CRD generator</name>
+
+  <properties>
+    <version.sundr>0.24.0</version.sundr>
+    <version.jackson>2.10.3</version.jackson>
+
+    <!-- Testing Dependencies -->
+    <version.junit-jupiter>5.5.0</version.junit-jupiter>
+    <version.junit-platform>1.4.0</version.junit-platform>
+    <version.assertj>3.11.1</version.assertj>
+
+    <!-- Plugins -->
+    <version.maven-compiler-plugin>3.8.0</version.maven-compiler-plugin>
+    <version.maven-invoker-plugin>3.1.0</version.maven-invoker-plugin>
+    <version.maven-failsafe-plugin>3.0.0-M3</version.maven-failsafe-plugin>
+    <version.maven-surefire-plugin>3.0.0-M3</version.maven-surefire-plugin>
+    <version.maven-shade-plugin>3.0.0</version.maven-shade-plugin>
+
+    <!-- Formatting Plugins -->
+    <version.formatter-maven-plugin>2.12.2</version.formatter-maven-plugin>
+    <version.impsort-maven-plugin>1.4.1</version.impsort-maven-plugin>
+
+  </properties>
```

Comment:
We should move>merge these with the parent's

https://github.com/fabric8io/kubernetes-client/blob/4404d62b99f6d3153104fec521704fa3816ffbd2/pom.xml#L77

Reply:
Done.